lib/logstash/devutils/rspec/shared_examples.rb

```diff
@@ -0,0 +1,23 @@
+require 'rspec/wait'
+
+RSpec.shared_examples "an interruptible input plugin" do
+  describe "#stop" do
+    let(:queue) { SizedQueue.new(20) }
+    subject do
+      s = described_class.new(config)
```

Comment:
I see the point, my purpose with this was to reduce the overhead of using this shared_example everywhere. I wonder if there's a way to make described_class.new(config) the default, but allow overriding this